source/postprocess/gravity_point_values.cc

```diff
@@ -421,12 +433,13 @@ namespace aspect
       {
         prm.enter_subsection ("Gravity calculation");
         {
-          prm.declare_entry ("Sampling scheme", "map",
-                             Patterns::Selection ("map|list"),
-                             "Choose the sampling scheme. A map will produce a grid of "
-                             "equally spaced points between "
-                             "a minimum and maximum radius, longitude, and latitude. A "
-                             "list contains specific coordinates of the satellites.");
+          prm.declare_entry ("Sampling scheme", "uniform distribution",
+                             Patterns::Selection ("uniform distribution|list of points|map|list"),
```

Comment:
Since you allow the old names here, also allow them in the `parse_parameters` function